WAF documentation update to reflect UI changes. (#223)
tigera/docs: calico-cloud/threat/web-application-firewall.mdx

### Configure a cluster for WAF

#### Step 1: Enable WAF in your cluster

On the Manager UI, click **Threat Defense -> Web Application Firewall -> Configure Web Application Firewall**.

> <img

Reviewer: I think the docs usually don't include screenshots, unless the screen is complicated enough that text describing it isn't enough. See this slack [message](https://tigera.slack.com/archives/C02BRQ66W8P/p1674501295758419?thread_ts=1674492642.842479&cid=C02BRQ66W8P) from @malavikabala

Author: I have removed the screenshot.

### Configure a cluster for WAF

> #### Step 1: Enable WAF in your cluster

Reviewer: Thinking about it, we don't want to update the CC docs in this release, because these UI changes will not be included in CC until it starts using 3.16 in about 3-4 months time?
@ctauchen if @bartolini changes the CC docs in this PR, they'll be what the customer starts seeing when it gets merged correct?

Reviewer 2: Based on where they're located right now, they'll land in CC/3.16.

Author: I have reverted docs changes for CC.

On the Manager UI, click **Threat Defense -> Web Application Firewall -> Configure Web Application Firewall**.

**Alternatively you can enable WAF manually by configuring ApplicationLayer CRD.**

> Create or update the [ApplicationLayer](https://docs.tigera.io/reference/installation/api#operator.tigera.io/v1.ApplicationLayer) resource to include the `webApplicationFirewall` section of the file. Ensure value of the field is set to `Enabled`.

Reviewer: ```suggestion
Create or update the [ApplicationLayer](https://docs.tigera.io/reference/installation/api#operator.tigera.io/v1.ApplicationLayer) resource to include the `webApplicationFirewall` field, and ensure it is set to `Enabled`.
```

  webApplicationFirewall: Enabled
```

#### Disabling WAF

> To disable WAF update the [ApplicationLayer](https://docs.tigera.io/reference/installation/api#operator.tigera.io/v1.ApplicationLayer) resource to include the `webApplicationFirewall` section of the file and ensure value of the field is set to `Disabled`.

Reviewer: ```suggestion
To disable WAF update the [ApplicationLayer](https://docs.tigera.io/reference/installation/api#operator.tigera.io/v1.ApplicationLayer) resource to include the `webApplicationFirewall` field, and ensure it is set to `Disabled`.
```

  webApplicationFirewall: Enabled
```

:::note

> WAF can be disabled in your cluster by setting the value of the `webApplicationFirewall` field to `Disabled`.

Reviewer: I would show an example of the yaml here with the field set to Disabled, I'm not sure some users will know they're meant to be changing the `ApplicationLayer` CR.

Also, the bug [here](https://tigera.atlassian.net/browse/RS-704) says that a brief description of how to disable using the UI would be handy, because it's not completely obvious how to do it.

Author: I have included more explicit instructions on how to disable WAF.
